queries/dune_v2/eth_spent.sql

```diff
@@ -0,0 +1,9 @@
+-- V2: https://dune.com/queries/1320174
```

Comment:
Same missing `num_transactions` and also missing a inner join with `ethereum.transactions`

Reply:
If this needs the said accuracy and we can live with a 10-15~ sec longer execution time I think we should continue without the batches table.

Reply:
Maybe I will jut include both equivalent queryies in this query file and leave one commented out with some comment on the performance and reliability tradeoffs.

Reply:
That's also a good idea!